plugins.d/Lets_Encrypt/dehydrated-wrapper

```diff
@@ -158,21 +162,24 @@ restart_servers() {
     for servicename in $@; do
         info "(Re)starting $servicename"
         systemctl restart $servicename | tee -a $LOG
-        [ "${PIPESTATUS[0]}" -eq 0 ] || EXIT_CODE=1
+        [[ "${PIPESTATUS[0]}" -eq 0 ]] || EXIT_CODE=1
     done
 }
 
 clean_finish() {
     # warning: do NOT use 'fatal' in this func as it will cause an inescapable recursive loop
```

Comment:
if you add `EXIT` to the trap signals in L216 then `fatal` can just `exit 1` and can be used here. the `EXIT` trap is executed exactly once